primitives/src/crh/poseidon/parameters/tweedle.rs

```diff
@@ -0,0 +1,487 @@
+use crate::crh::{
+    PoseidonParameters,
+    FieldBasedHashParameters, PoseidonHash, batched_crh::PoseidonBatchHash,
+    PoseidonQuinticSBox,
+};
+use algebra::fields::tweedle::Fq as tweedleFq;
+use algebra::fields::tweedle::Fr as tweedleFr;
+
+use algebra::biginteger::BigInteger256;
+use algebra::field_new;
+
+#[derive(Clone)]
+pub struct TweedleFqPoseidonParameters;
+
+impl FieldBasedHashParameters for TweedleFqPoseidonParameters {
+    type Fr = tweedleFq;
+    const R:usize = 2;  // The rate of the hash function
+}
+
+impl PoseidonParameters for TweedleFqPoseidonParameters {
+
+    const T:usize = 3;  // Number of S-Boxes
+    const R_F:i32 = 4;  // Number of full rounds (for each of the two full rounds)
+    const R_P:i32 = 53; // Number of partial rounds
+
```

Comment:
I would recommend to take the same parameters as in the [updated Poseidon paper](https://eprint.iacr.org/2019/458) on   the IACR preprint archive. In any approximately 256 Bit sized prime field, they recommend `R_F=4` and `R_P = 57` as number of rounds for the x^5-POSEIDON-128.

Reply:
The new parameters are already integrated in the branch [UpdatePoseidonParameters](https://github.com/HorizenOfficial/ginger-lib/tree/UpdatedPoseidonParameters).